src/pages/VouchingPage/NewNominationModal.tsx

```diff
@@ -59,69 +52,209 @@ export const NewNominationModal = ({
+            'grid-template-rows': 'auto auto',
+            'column-gap': '$lg',
+            'row-gap': '$xl',
           }}
-          fullWidth
-        />
-      </div>
-    </FormModal>
+        >
+          <Box
+            css={{
+              display: 'flex',
+              flexDirection: 'column',
+              alignItems: 'center',
+              width: '100%',
+            }}
+          >
+            <FormLabel
+              htmlFor="name"
+              css={{
+                lineHeight: '1.3',
+                color: '$text',
+                fontSize: '$4',
+                fontFamily: 'Inter',
+                fontWeight: '$bold',
+                textAlign: 'center',
+                m: '0 0 $sm',
+              }}
+            >
+              Name
+            </FormLabel>
+            <Controller
+              name={'name'}
+              defaultValue={intialValues.name}
+              rules={{
+                required: 'Name must be at least 3 characters long.',
+                minLength: {
+                  value: 3,
+                  message: 'Name must be at least 3 characters long.',
+                },
+              }}
+              control={control}
+              render={({ field }) => (
+                <TextField
+                  css={{ height: '48px', width: '100%' }}
+                  id="name"
+                  {...field}
+                />
+              )}
+            />
+          </Box>
+          <Box
+            css={{
+              display: 'flex',
+              flexDirection: 'column',
+              alignItems: 'center',
+              width: '100%',
+            }}
+          >
+            <FormLabel
+              htmlFor="address"
+              css={{
+                lineHeight: '1.3',
+                color: '$text',
+                fontSize: '$4',
+                fontFamily: 'Inter',
+                fontWeight: '$bold',
+                textAlign: 'center',
+                m: '0 0 $sm',
+              }}
+            >
+              ETH Address
+            </FormLabel>
+            <Controller
+              name={'address'}
+              defaultValue={intialValues.address}
+              control={control}
+              rules={{
+                validate: value =>
+                  ethers.utils.isAddress(value) || 'Invalid address',
+              }}
+              render={({ field }) => (
+                <TextField
+                  css={{ height: '48px', width: '100%' }}
+                  id="address"
+                  {...field}
+                />
+              )}
+            />
+          </Box>
+          <Box
+            css={{
+              'grid-column': '1 / -1',
+              display: 'flex',
+              flexDirection: 'column',
+              alignItems: 'center',
+              width: '100%',
+            }}
+          >
+            <FormLabel
+              htmlFor="description"
+              css={{
```

Comment:
Even though this isn't a variant anymore, it is still the same the 3 times it's used in this file, so it would be good to pull it out into a variable to reduce duplication.

Reply:
added labelStyles const